src/validators/__tests__/validateHobbies.test.ts

```diff
@@ -0,0 +1,29 @@
+import { describe, it, expect } from 'vitest'
+
+import { validateHobbies } from '@/validators/studentsValidators'
+
+describe('validateHobbies', () => {
+  it('should return true for valid hobby strings', () => {
+    expect(validateHobbies('Reading')).toBe(true)
+    expect(validateHobbies('   Cooking, coding    ')).toBe(true)
+    expect(validateHobbies('Photography,    Hiking, Swimming')).toBe(true)
+    expect(validateHobbies('1, 2,   3')).toBe(true)
+    expect(validateHobbies('Writing, 4, CoDIng')).toBe(true)
+    expect(validateHobbies('')).toBe(true)
+    expect(validateHobbies('     ')).toBe(true)
+  })
+
+  it('should return false for invalid hobby strings', () => {
+    expect(validateHobbies(',Reading')).toBe(false)
```

Comment:
imo this function should be immune to this type of action. Let's improve it and just remove trailing commas